x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/manual_configuration.tsx

```diff
@@ -40,9 +59,104 @@ export const ManualConfiguration: React.FC<ManualConfigurationProps> = ({
   const closePopover = () => {
     setPopover(false);
   };
-
+  const { selectedConnector, rawName } = useValues(NewConnectorLogic);
   const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
   const [flyoutContent, setFlyoutContent] = useState<'manual_config' | 'client'>();
+  const getCodeSnippet = (): string => {
+    const connectorInfo: NativeConnector | undefined = selectedConnector?.serviceType
+      ? NATIVE_CONNECTOR_DEFINITIONS[selectedConnector.serviceType]
+      : undefined;
+    if (!connectorInfo) {
+      return '';
+    }
+
+    const dynamicConfigValues = Object.entries(
+      connectorInfo.configuration as ConnectorConfiguration
+    )
+      .map(([key, config]) => {
+        const defaultValue = config ? JSON.stringify(config.value) : null;
+        return `    "${key}": ${defaultValue}`;
+      })
+      .join(',\n');
+    const CONSOLE_SNIPPET = dedent` # Example of how to create a ${connectorInfo?.name} connector using the API
+# This also creates related resources like an index and an API key.
+# This is an alternative to using the UI creation flow.
+
+# 1. Create an index
+PUT /connector-${rawName}
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 3,
+      "number_of_replicas": 2
+    }
+  }
+}  
+# 2. Create a connector
+PUT _connector/${rawName}
```

Comment:
Uhh `rawName` is very misleading, can we change it to `connectorId`??

`name` implies it's a freeform string (like something that could include spaces, etc.) but since we’re using it as an ES ID, it has to follow a strict pattern. I think it’d be clearer to indicate in the code that this is actually an id variable.

Reply:
It makes sense to use the rawName to get the same autogenerated name one we choose a connector. But it's true that we are not checking if the users once the name is generated they can add extra spaces and non accepted characters. So I'd keep using this `rawName` but we should check any modification in the Connectors name in a new ticket

Reply:
Tracking this issue here: https://github.com/elastic/search-team/issues/8560